### PR TITLE
Update supported-platforms.md

### LINKED ISSUE #25340: Version of Windows for running ASP.NET CORE gRPC services in IIS
The documentation says that in order to host gRPC services on IIS, "IIS requires .NET 5 and Windows 10 Build 20300.1000 or later."
It also mention that it might need a Windows insider build. I could not find a version 20300.1000 here: https://blogs.windows.com/blog/tag/windows-insider-program/

As such the documentation regarding OS requirement is quite vague and I would assume outdated as well.
It would be good to update the documentation to list the minimum version of Generally Available OS.
-> Is the feature to host gRPC services on IIS now currently supported on a Generally Available version of Windows 10 (e.g. 21H2 build 19044.1586)?
-> Same question but for a server version of Windows (so that it can be used on production grade environment)?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 76d3f0ff-f65f-97d2-940f-4509b0d425bd
* Version Independent ID: ea62de0b-d7e5-e543-b892-a587d6e9dc84
* Content: [gRPC on .NET supported platforms](https://docs.microsoft.com/en-us/aspnet/core/grpc/supported-platforms?view=aspnetcore-6.0)
* Content Source: [aspnetcore/grpc/supported-platforms.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/grpc/supported-platforms.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-grpc**
* GitHub Login: @JamesNK
* Microsoft Alias: **jamesnk**

### PR DESCRIPTION
**Pull request by JamesNK**
Fixes #25340

### CODE REVIEW

File: aspnetcore/grpc/supported-platforms.md [hunk 1]
```diff
@@ -56,11 +56,9 @@ All built-in ASP.NET Core servers are supported.
 > * IIS&dagger;
 > * HTTP.sys&Dagger;
 
-&dagger;IIS requires .NET 5 and Windows 10 Build 20300.1000 or later.
```
Comment (guardrex) [inline aspnetcore/grpc/supported-platforms.md:59]:
```suggestion
&dagger;IIS requires .NET 5 and Windows 10 Build 20300.1000 or later.  
```

**Comment by guardrex** [comment]
GH is having real problems this morning ... lots of 500's 💥.

**Comment by guardrex** [comment]
@JamesNK ... FYI on those two spaces: It's a markdown gesture to insert a bare return `<br>`. If you'd like me to stack those two lines into a paragraph tho, let me know 👂 ... I'll submit a patch PR to 🔪 them back out.

**Comment by JamesNK** [comment]
I didn’t know markdown did that.

Adding them back is correct. Thanks

**Pull request merged**